### DISCUSSION TITLE
Matching all markers to the map view not work

### DISCUSSION THREAD

**Discussion by F50Ci37Y** [tomik23/react-leaflet-examples#15] [Announcements]
It's not working, I clone react-leaflet-examples and change center and zoom and not working

**Comment by tomickigrzegorz** [comment]
Well, look, and it works for me 😀
I can't do anything without a working example + code.

PS. I transfer the problem to the "Discussions" tab.

**Reply by F50Ci37Y** [reply]
> Well, look, and it works for me 😀 I can't do anything without a working example + code.
> 

Yes, I was wrong.
I use `useMap` and work fine.
``` javascript
function ControllingGroup() {
  const map = useMap();
  const bounds = new L.LatLngBounds();
  map.eachLayer((layer) => {
    if (layer instanceof L.FeatureGroup) {
      bounds.extend(layer.getBounds());
    }
  });
  if (bounds.isValid()) {
    map.fitBounds(bounds);
  }
  return null;
}

```
